tests/tx/test_verification.py

```diff
@@ -265,6 +300,46 @@ def test_merge_mined_block_verify(self) -> None:
         # MergeMinedBlock methods
         verify_pow_wrapped.assert_called_once()
 
+    def test_merge_mined_block_validate_full(self) -> None:
```

Comment:
It's missing `verify_weight()` and `verify_reward()`. They are both called from `verify_basic()`.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/824/commits/408b1d147c6e83216782bd3311bfa599604c6508

Reply:
Thanks!